libc/include/llvm-libc-types/float128.h

```diff
@@ -9,8 +9,6 @@
 #ifndef __LLVM_LIBC_TYPES_FLOAT128_H__
 #define __LLVM_LIBC_TYPES_FLOAT128_H__
 
-#include <include/llvm-libc-macros/float-macros.h> // LDBL_MANT_DIG
```

Comment:
`LDBL_MANT_DIG` is definitely used below; something seems off to refer to it without include the previous declaration of it. Like an IWYU violation.

Reply:
Maybe we could use `#include <../llvm-libc-macros/float-macros.h>`? Does that even work cross-platform? I'd say currently being able to include `math.h` without the compiler erroring on a nonexistant path is superior to a IWYU violation currently.

Reply:
> Does that even work cross-platform?

/me shrugs in Linux.

@lntue 's suggestion to fix our include paths to match how the headers will be installed sounds like it would straighten this out.

Reply:
Just did it, should work fine.

Reply:
Can we now remove `include/` from the `#include <>` preprocessor statements elsewhere now as a result of your cmake changes?

```sh
$ grep -rn -e '#include "include/' -e '#include <include/' libc
```

Reply:
Honestly, I think it should still be explicit when used outside of `include` itself. Keeps it consistent I feel.

Reply:
Consistency is good.  So is brevity; `#include "include/...` looks silly to me and if your cmake change allows us to omit the extra include everywhere, I think the result would still be consistent, and then we don't have an undocumented convention that will get out of sync again.

Reply:
Alright, change incoming from `sed -i 's/include "include/include "/`